defaults/main.yml

```diff
@@ -379,7 +356,7 @@ checkmk_agent__plugin_nginx_servers:
 # .. envvar:: checkmk_agent__git_repo [[[
 #
 # Check_MK agent source repository.
-checkmk_agent__git_repo: 'http://git.mathias-kettner.de/check_mk.git'
+checkmk_agent__git_repo: 'https://git.mathias-kettner.de/check_mk.git'
```

Comment:
Nice catch. This wasn't possible when I added this feature, as the certificate was not valid for the `git` subdomain. Finally they fixed it (with Let's encrypt :smile:)